src/query-builder.js

```diff
@@ -19,10 +19,10 @@ module.exports = {
   },
   build(query, widgetQuery) {
     const params = {
-      query: `"${query.text}",${fields.language}:(english|en)`,
+      query: `"${query.text}"`,
     };
     if (query.date) {
-      params.filter = `${fields.publication_date}>${moment(query.date.from).format(ISO_8601)},${fields.publication_date}<${moment(query.date.to).format(ISO_8601)}`;
+      params.filter = `${fields.publication_date}>${moment(query.date.from).format(ISO_8601)},${fields.publication_date}<${moment(query.date.to).format(ISO_8601)},${fields.language}:(english|en)`;
```

Comment:
i think its probably mostly fine since `.date` is always present, but we should probably do something like:
```
params.filter = `${fields.language}:(english|en)`
if (query.date) {
  params.filter = [
    params.filter,
   `${fields.publication_date}>${moment(query.date.from).format(ISO_8601)}`,
   `${fields.publication_date}<${moment(query.date.to).format(ISO_8601)}`
 ].join(',');
}
```